packages/components/nodes/documentloaders/Cheerio/Cheerio.ts

```diff
@@ -161,17 +165,17 @@ class Cheerio_DocumentLoaders implements INode {
             if (process.env.DEBUG === 'true') options.logger.info(`pages: ${JSON.stringify(pages)}, length: ${pages.length}`)
             if (!pages || pages.length === 0) throw new Error('No relative links found')
             for (const page of pages) {
-                docs.push(...(await cheerioLoader(page)))
+                docs.push(...((await cheerioLoader(page)) || []))
             }
             if (process.env.DEBUG === 'true') options.logger.info(`Finish ${relativeLinksMethod}`)
         } else if (selectedLinks && selectedLinks.length > 0) {
             if (process.env.DEBUG === 'true')
                 options.logger.info(`pages: ${JSON.stringify(selectedLinks)}, length: ${selectedLinks.length}`)
             for (const page of selectedLinks.slice(0, limit)) {
-                docs.push(...(await cheerioLoader(page)))
+                docs.push(...((await cheerioLoader(page)) || []))
```

Comment:
instead of placing the `[]`, I think we should just modify the `cheerioLoader` function to return `[]` when `catch (err)`

Reply:
I agree with you. I've added a fix as you suggested.